packages/artillery-plugin-memory-inspector/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "artillery-plugin-memory-inspector",
+  "version": "1.0.0",
+  "description": "Inspect the memory of processes running together with your load tests!",
+  "main": "index.js",
+  "scripts": {
+    "test": "tap ./test/*.spec.mjs --no-coverage --color --timeout 300"
```

Comment:
Can you please elaborate more on why we need a custom timeout?

Reply:
Tap documentation doesn't state the default timeout but it's not that big IIRC. As these tests take a little bit to run - about 1 min - (since I wanted to generate enough metrics), I added a bigger timeout. A few of the other tests also adopt 300, so I went with that value.

Reply:
Thanks for the explanation!